deps/picotls/include/picotls.h

```diff
@@ -1157,6 +1163,10 @@ ptls_context_t *ptls_get_context(ptls_t *tls);
  * updates the context of a connection. Can be called from `on_client_hello` callback.
  */
 void ptls_set_context(ptls_t *tls, ptls_context_t *ctx);
+/**
+ * get the sign ctx
+ */
+void *ptls_get_sign_ctx(ptls_t *tls);
```

Comment:
```suggestion
void *ptls_get_sign_context(ptls_t *tls);
```
So that the name would in alignment with "ptls_get_context" that we already have.